<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PI</class>
 <widget class="QMainWindow" name="PI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="xoLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="yoLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>xs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="xsLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>ys</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ysLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>x0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="x0LineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>y0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="y0LineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>xo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="xoLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>yo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="yoLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>xn</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="xnLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>yn</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ynLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="xnLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ynLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="10" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="findAxesButton">
        <property name="text">
         <string>Find axes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="findCameraButton">
        <property name="text">
         <string>Find camera</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="moveToStartPositionButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Move to start position</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startScanningButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Start scanning</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelScanningButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Cancel scanning</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
      <item>
       <widget class="QLabel" name="x0Label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="y0Label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="xsLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ysLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="15" column="0">
     <widget class="QTextBrowser" name="logs"/>
    </item>
    <item row="9" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Path to save pictures</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pathLine">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browseButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>x0LineEdit</tabstop>
  <tabstop>y0LineEdit</tabstop>
  <tabstop>xnLineEdit</tabstop>
  <tabstop>ynLineEdit</tabstop>
  <tabstop>xsLineEdit</tabstop>
  <tabstop>ysLineEdit</tabstop>
  <tabstop>xoLineEdit</tabstop>
  <tabstop>yoLineEdit</tabstop>
  <tabstop>findAxesButton</tabstop>
  <tabstop>moveToStartPositionButton</tabstop>
  <tabstop>startScanningButton</tabstop>
  <tabstop>cancelScanningButton</tabstop>
  <tabstop>logs</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
